.golangci.yml

```diff
@@ -30,6 +30,8 @@ issues:
       text: "err113: do not define dynamic errors, use wrapped static errors instead"
     - path: _test\.go
       text: "error returned from external package is unwrapped"
+    - path: _test\.go
+      text: "type assertion must be checked"
```

Comment:
why do you need to to that? you didn't change any _test\.go file?

Reply:
Existing linting issues from make lint